File: Ejercicio5-condicionales.cpp
/*5. Escriba un programa que lea de la entrada estándar un carácter e indique en la salida estándar si el carácter es una vocal minúscula o no.*/

#include<iostream>
using namespace std;
int main (){
    char letra;

    cout<<"Digite una vocal: ";
    cin>>letra;

    switch(letra){
        case 'a':
        case 'e':
        case 'i':
        case 'o':
        case 'u': cout<<"\nEs una vocal minuscula";break;
        default: cout<<"NO es una vocal minuscula";break;
    }
    return 0;
}

File: Comparativa Python/Ejercicio5-Condicionales.py
"""5. Escriba un programa que lea de la entrada estándar un carácter e indique
en la salida estándar si el carácter es una vocal minúscula o no."""

caracter = input('Digite un caracter: ')
if caracter == 'a' or caracter == 'e' or caracter == 'i' or caracter == 'o' or caracter == 'u':
    print('Es una vocal minúscula.')
else:
    print('El caracter digitado NO es una vocal minúscula')
